_sass/kroxylicious.scss

```diff
@@ -33,6 +33,14 @@ $navbar-dark-disabled-color: rgba($white, 0.45);
 // Custom bootstrap variables must be set or imported *before* bootstrap.
 @import "./bootstrap/scss/bootstrap";
 
+body {
+  height: fit-content!important;
+}
+
+.krx-content * {
+  scroll-margin-top: 5rem!important;
```

Comment:
The navbar is actually closer to `3.5rem` but once I accounted for drop-shadow and line spacing `5rem` ended up being what looked best. I used `rem` and not `em` so that everything had consistent spacing from the top of the page, otherwise smaller headings would have been closer to the top and larger headings would have been further. I avoided fixed `px`, `vh`, and `%` units because it broke things in new and interesting ways on phone screens.

Reply:
Ohh that was a lot more funky than I expected thank you!

Reply:
Yeah, I'm so glad I didn't try that rabbit hole myself!